drivers/sensor/openrocket/conversion.py

```diff
@@ -18,13 +18,13 @@
 
 
 OREvent = namedtuple("OREvent", ['time', 'event'])
-type Reason = str
```

Comment:
Truly a tragedy that some people are forced to live with old versions off python :pensive:

Reply:
Big L